test/src/org/exist/storage/MoveTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ *  eXist Open Source Native XML Database
+ *  Copyright (C) 2001-2016 The eXist Project
+ *  http://exist-db.org
+ *
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public License
+ *  as published by the Free Software Foundation; either version 2
+ *  of the License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.exist.storage;
+
+import org.exist.Database;
+import org.exist.EXistException;
+import org.exist.TestUtils;
+import org.exist.collections.Collection;
+import org.exist.collections.IndexInfo;
+import org.exist.dom.QName;
+import org.exist.dom.persistent.*;
+import org.exist.indexing.StructuralIndex;
+import org.exist.storage.txn.Txn;
+import org.exist.util.Configuration;
+import org.exist.util.DatabaseConfigurationException;
+import org.exist.xmldb.XmldbURI;
+import org.exist.xquery.NodeSelector;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author wolf
+ *
```

Comment:
Is this a test you have copied from somewhere? or is this a test you have written?